platform-includes/session-replay/setup/javascript.mdx

```diff
@@ -31,6 +31,14 @@ setTimeout(() => {
   throw new Error("Sentry Test Error");
 });
 ```
+### PII & Privacy Considerations
+
+Personally identifiable information (PII), and privacy are important considerations when enabling session replays. There are several ways in which Sentry helps you avoid collecting PII:
+- [Masking](/platforms/javascript/session-replay/privacy/#masking) which replaces the text content with something else (the default behavior being to replace each character with a *).
+- Making [Network request and response bodies and headers](/javascript/session-replay/privacy//javascript/session-replay/privacy/#network-request-and-response-bodies-and-headers/) an opt-in feature, because the best way to avoid getting PII into Sentry is by not adding URLs of endpoints that may contain PII.
```

Comment:
```suggestion
- Making [Network request and response bodies and headers](/platforms/javascript/session-replay/privacy/#network-request-and-response-bodies-and-headers) an opt-in feature, because the best way to avoid getting PII into Sentry is by not adding URLs of endpoints that may contain PII.
```